modules/configuring-firewall.adoc

```diff
@@ -51,6 +51,10 @@ CDN hostnames, such as `cdn01.quay.io`, are covered when you add a wildcard entr
 |===
 |URL | Port | Function
 
+|`cloud.redhat.com`
+|443, 80
+|Required for Telemetry
```

Comment:
Installer QE can't ack this, we don't test Telemetry

Reply:
I will remove this then?

Reply:
sorry, I don't know, we don't test this.

Reply:
sorry, I don't know, we don't test this.